aviary/validation_cases/benchmark_tests/test_bench_GwGm.py

```diff
@@ -37,33 +37,33 @@ def test_bench_GwGm(self):
         # There are no truth values for these.
         assert_near_equal(
             prob.get_val(Mission.Design.GROSS_MASS, units='lbm'),
-            173806.9409,
+            171044.035,
             tolerance=rtol,
         )
 
         assert_near_equal(
             prob.get_val(Aircraft.Design.OPERATING_MASS, units='lbm'),
-            95620.2190,
+            94952.152,
             tolerance=rtol,
         )
 
         assert_near_equal(
             prob.get_val(Mission.Summary.TOTAL_FUEL_MASS, units='lbm'),
-            42186.7219,
+            40091.883,
```

Comment:
Interesting that the result of doubling the fuselage drag is that the gasp mission consumes about 2k lbs more fuel.